app/views/home/index.html.erb

```diff
@@ -1 +1,6 @@
-<%= render partial: "signup_form" %>
+<h1>￼Connecting businesses, communities and the tech talent pipeline</h1>
+
+<div class="links">
+  <%= link_to "Learn More", learn_more_index_path %>
+  <%= link_to "Sign Up", signup_path %>
+</div>
```

Comment:
Added Learn more and sign up as links. I figured whomever is doing the html and css can style these to look like buttons